scheduler/src/cook/mesos/api.clj

```diff
@@ -551,6 +557,108 @@
                           cheshire/generate-string)))
       ring.middleware.json/wrap-json-params))
 
+
+
+(defn retries
+  "Returns the number of retries for the given job-id, if the current
+  user is authorized to view it."
+  [conn fid request job-id]
+  (let [user (:authorization/user request)
+        job   (fetch-job-map (db conn)
+                             fid
+                             job-id) 
+        authorized?  (and job
+                          (is-authorized? user :read job))]
+    (log/info "[retries] User" user
+              "asked how many retries there are for job-id" job-id)
+    (if (not authorized?)
+      (do
+        (log/info "[retries] User" user 
+                  (if-not job
+                    "asked about a nonexistent job ID; denying."
+                    (str "is not authorized to view job ID " job-id "; denying") ))
+        {:status  404
+         :headers {"Content-Type" "application/json"}
+         :body    (str "Job ID " job-id " not found.")})
+      {:status  200
+       :headers {"Content-Type" "application/json"}
+       :body    (str (:max_retries job))})))
+
+
+(defn set-retries!
```

Comment:
Same comment as in shares, please use [liberator ](http://clojure-liberator.github.io/liberator/) to help componentize this.

Also, please use query parameter form we use for submitting jobs instead of embedding the job id in the url to be consistent.